### PR TITLE
2016.09.13 cherry picks

### PR DESCRIPTION
**Pull request by mdlinville**
Cherry-picks from 9-13. Please disregard PR #26572 which was opened erroneously.

@thaJeztah @SvenDowideit @vdemeester PTAL. CC @riyazdf

### CODE REVIEW

File: docs/reference/api/docker_remote_api.md [hunk 1]
```diff
@@ -111,6 +111,16 @@ Running `docker rmi` emits an **untag** event when removing an image name.  The
...
 
+### v1.25 API changes
+
+[Docker Remote API v1.25](docker_remote_api_v1.25.md) documentation
+
+* `POST /containers/create` now takes `AutoRemove` in HostConfig, to enable auto-removal of the container on daemon side when the container's process exits.
+* `GET /containers/json` and `GET /containers/(id or name)/json` now return `"removing"` as a value for the `State.Status` field if the container is being removed. Previously, "exited" was returned as status.
+* `GET /containers/json` now accepts `removing` as a valid value for the `status` filter.
+* `DELETE /volumes/(name)` now accepts a `force` query parameter to force removal of volumes that were already removed out of band by the volume driver plugin.
+* `POST /containers/create/` and `POST /containers/(name)/update` now validates restart policies.
```
Comment (thaJeztah) [inline docs/reference/api/docker_remote_api.md]: Oops, looks like you forgot to remove this commit here

**Comment by thaJeztah** [comment]
One note, otherwise LGTM

File: docs/reference/api/docker_remote_api.md (hunk 1 above)
Comment (mstanleyjones) [inline docs/reference/api/docker_remote_api.md]: Wow, how did I miss it? I've taken it out again and re-pushed.

Comment (thaJeztah) [inline docs/reference/api/docker_remote_api.md]: no worries, that's where reviews are for aren't they? 👍

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Pull request merged**